app/controllers/teams_controller.rb

```diff
@@ -27,10 +27,13 @@ def team_emp
   emp_teams=team.employees
   emps=[]
   emp_teams.each do |emp|
+    date=EmployeeTeam.where(employee_id:emp.id)
```

Comment:
In this case, we will want to add another condition for selecting the EmployeeTeam. We need to check both that the employee_id matches the id of the employee and that the end_date is nil, because it is possible for multiple EmployeeTeam records to exist for a specific employee_id. The record that has no end_date represents the employee's current team membership

Reply:
i added end date clause on where and next to skip loop if date empty